cmd/debug.go

```diff
@@ -64,10 +64,14 @@ func Debug(ctx *cli.Context) {
 
 	if configured {
 		fmt.Printf("Config file: %s\n", c.File)
-		fmt.Printf("API Key: %s\n", c.APIKey)
+		if c.APIKey != "" {
+			fmt.Printf("API Key: %s\n", c.APIKey)
+		} else {
+			fmt.Println("API Key: Please set your API Key to access all of the CLI features")
+		}
 	} else {
 		fmt.Println("Config file: <not configured>")
-		fmt.Println("API Key: <not configured>")
+		fmt.Println("API Key: Please set your API Key to access all of the CLI features")
```

Comment:
Let's leave this with the `<not configured>` text as it's on the case where the CLI is not configured at all and it makes more sense.